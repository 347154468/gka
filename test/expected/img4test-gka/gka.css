.gka-animation {
    width: 50px;
    height: 50px;

    background-repeat: no-repeat;
    background-position: center center;
    background-size: contain;;

    -webkit-animation: gka_keyframes 0.32s steps(1) infinite;
}

@-webkit-keyframes gka_keyframes {
    12.50%{
        background-image: url("img/gka-1.png");
    }

    25.00%{
        background-image: url("img/gka-2.png");
    }

    37.50%{
        background-image: url("img/gka-3.png");
    }

    50.00%{
        background-image: url("img/gka-4.png");
    }

    62.50%{
        background-image: url("img/gka-5.png");
    }

    75.00%{
        background-image: url("img/gka-3.png");
    }

    87.50%{
        background-image: url("img/gka-2.png");
    }

    100.00%{
        background-image: url("img/gka-6.png");
    }

    99.99999%{
        opacity: 1;
    }

    100%{
        opacity: 0;
    }
}
