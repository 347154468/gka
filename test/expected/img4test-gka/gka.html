
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>gka-animation-preview</title>
    <style>
        body {
            background-color: #000;
        }
    </style>
    <link href="./gka-gka.css" rel="stylesheet" type="text/css">
</head>
<body>
    <div class="gka-animation gka-animation"></div>
</body>
</html>
